server/chat-plugins/trivia.ts

```diff
@@ -1793,19 +1770,22 @@ const triviaCommands: ChatCommands = {
 
 		const submissions = triviaData.submissions;
 		if (!submissions) return this.sendReply(this.tr`No questions await review.`);
-		const submissionsLen = submissions.length;
-		if (!submissionsLen) return this.sendReply(this.tr`No questions await review.`);
-
-		let buffer = `|raw|<div class="ladder"><table>` +
-			`<tr><td colspan="6"><strong>${Chat.count(submissionsLen, "</strong> questions")} awaiting review:</td></tr>` +
-			`<tr><th>#</th><th>${this.tr`Category`}</th><th>${this.tr`Question`}</th><th>${this.tr`Answer(s)`}</th><th>${this.tr`Submitted By`}</th></tr>`;
 
+		let innerBuffer = '';
 		let i = 0;
-		while (i < submissionsLen) {
-			const entry = submissions[i];
-			buffer += `<tr><td><strong>${(++i)}</strong></td><td>${entry.category}</td><td>${entry.question}</td><td>${entry.answers.join(", ")}</td><td>${entry.user}</td></tr>`;
+		for (const category in submissions) {
+			for (const entry of submissions[category]) {
```

Comment:
```suggestion
			for (const [i, entry] of submissions[category].entries()) {
```

Reply:
We need `i` outside of the loop, on line 1785. (It's the total number of questions in all categories that await review.)

Reply:
Call it index or something, then?

Reply:
Or tbh maybe just rename the other variable, since "i" seems unclear for that.

Reply:
Yes, I named it `total` instead.